src/js/dp/improve-this-page.js

```diff
@@ -34,6 +34,7 @@ function fetchFeedbackAPI(
       formHeader.innerHTML = feedbackMessage;
     })
     .catch((error) => {
+      // eslint-disable-next-line no-console
       console.error(error);
```

Comment:
I think we should be removing this console error really - what do you think?

Reply:
yep, the error is handled and the user is presented with some next steps/things to try